crates/bevy_color/src/hsla.rs

```diff
@@ -66,6 +66,24 @@ impl Hsla {
     pub const fn with_lightness(self, lightness: f32) -> Self {
         Self { lightness, ..self }
     }
+
+    /// Generate a deterministic but randomly distributed color from a provided `index`.
```

Comment:
```suggestion
    /// Generate a deterministic but quasi-randomly distributed color from a provided `index`.
    ///
    /// These colors are designed to be distributed evenly but randomly through color space.
```

Reply:
Remember to update the sibling to match :)

I'd also be happy with a wikipedia link here.